Makefile

```diff
@@ -161,6 +162,8 @@ else
 GINKGO_SKIP := upgrade
 endif
 
+GINKGO_FOCUS := live-iso
```

Comment:
/hold we cannot merge that else the other tests will not get triggered since ginkgo here focusing only on live-iso without condition, we should wait until the jjb https://gerrit.nordix.org/c/infra/cicd/+/13707 get merged then remove this line